gammapy/irf/exposure.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from ..data import SpectralCube
+from astropy.coordinates import SkyCoord, Angle
+
+import logging
+
+__all__ = [
+    'exposure_cube'
+]
+
+log = logging.getLogger(__name__)
+
+
+def exposure_cube(pointing,
+                  livetime,
+                  aeff2D,
+                  ref_cube):
+    """Exposure cube
+
+        Parameters
+        ----------
+        pointing : `~astropy.coordinates.SkyCoord`
+            pointing direction
+        livetime : `~astropy.units.Quantity`
+            livetime
+        aeff2D: 'EffectiveAreaTable2D'
+            effective area table
+        ref_cube: 'SpectralCube'
```

Comment:
Can you try running `python setup.py build_sphinx` and check the HTML API docs locally?
I think you have to reference the class from another file more explicitly for the cross-link to work, no?
And I think there has to be a space before the colon for the formatting to be parsed correctly by Sphinx.
```
       ref_cube : '~gammapy.spectrum.SpectralCube'
```